packages/concerto-cto/lib/printer.js

```diff
@@ -180,27 +180,27 @@ function declFromMetaModel(mm) {
         result += 'abstract ';
     }
     switch (mm.$class) {
-    case 'concerto.metamodel.AssetDeclaration':
+    case 'concerto.metamodel@1.0.0.AssetDeclaration':
```

Comment:
It would be convenient to have `concerto.metamodel@1.0.0` refactored somewhere to avoid touching all of these files when we need to bump the version in future. Is there any benefit to making it dynamic?